code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm

```diff
@@ -490,9 +515,10 @@
 	M.apply_effect(16 * removed, IRRADIATE, 0)
 
 /datum/reagent/aslimetoxin
-	name = "Docility Toxin"
-	id = "docilitytoxin"
-	description = "A corruptive toxin produced by slimes."
+	name = "Advanced Mutation Toxin"
```

Comment:
...Assumingly because something in the cherry-pick decided it needed to be changed, and I, not knowing Xenobio's new system very well, didn't catch it. Thanks.